wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -343,7 +344,7 @@ public void stopWorkspace(String workspaceId, Map<String, String> options)
             });
   }
 
-  public Set<String> getSupportedRecipes() {
+  Set<String> getSupportedRecipes() {
```

Comment:
Could you explain why this method should be package private but not public?

Reply:
Well, there are no other reasons than IntelliJ suggestion and absence of javadoc (I thought it is not important enough to be public )) ). If you think it is useful, I do not mind to revert and add javadocs as well.